### DISCUSSION TITLE
I can't select the Card Payment Method on my Medusa Frontend

### DISCUSSION THREAD

**Discussion by lfdadev** [medusajs/medusa#5966] [Q&A]
Hi! I installed the medusa project with custom nextjs frontend. I followed the guide to install medusa-stripe. I have deployed with correct domains and I set all env. 
When I try to select the Card Method Payment, I get this error.

Web: https://www.lafuerzadelalma.com/
Github: https://github.com/lfdadev

![image](https://github.com/medusajs/medusa/assets/151443436/320a13db-553c-4044-91f9-04d9280c954e)

![image](https://github.com/medusajs/medusa/assets/151443436/f28fc0a6-c468-4bf8-84d7-4068ebc5e6c5)

I didn't change nothing more. Any idea?

![image](https://github.com/medusajs/medusa/assets/151443436/1e3aa21a-3182-488d-a4b4-66e775ec3f88)

![image](https://github.com/medusajs/medusa/assets/151443436/219aed4a-c545-47ac-a841-6a7fddcf3bb1)

**Comment by davidFCDev** [comment]
I need some help. I'm so stucked here

**Comment by Kushal-starr** [comment]
@davidFCDev may be issue is with the webhook you setup. Try without the webhooks it will still work. Thanks!
